### PR TITLE
MM-48988 Log plugins version on install and on startup

### PR DESCRIPTION
**Pull request by zeraussiul**
#### Summary

On plugin install/activation, adds logs with plugin id and version numbers

Sample output logs from tests:
```
...
{"timestamp":"2022-12-18 22:36:08.727 -05:00","level":"debug","msg":"Installing plugin","caller":"app/plugin_install.go:405","plugin_id":"testplugin","version":"0.0.1"}
{"timestamp":"2022-12-18 22:36:08.727 -05:00","level":"debug","msg":"Installing plugin","caller":"app/plugin_install.go:405","plugin_id":"testplugin2","version":"1.2.3"}
{"timestamp":"2022-12-18 22:36:08.741 -05:00","level":"debug","msg":"Starting plugin","caller":"plugin/environment.go:333","plugin_id":"testplugin","version":"0.0.1"}
{"timestamp":"2022-12-18 22:36:08.751 -05:00","level":"debug","msg":"Starting plugin","caller":"plugin/environment.go:333","plugin_id":"testplugin2","version":"1.2.3"}
...
```

#### Ticket Link

- Fixes https://github.com/mattermost/mattermost-server/issues/21878

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: plugin/environment.go [hunk 1]
```diff
@@ -330,6 +330,8 @@ func (env *Environment) Activate(id string) (manifest *model.Manifest, activated
 		return nil, false, fmt.Errorf("unable to start plugin: must at least have a web app or server component")
 	}
 
+	mlog.Debug("Starting plugin", mlog.String("plugin_id", pluginInfo.Manifest.Id), mlog.String("version", pluginInfo.Manifest.Version))
```
Comment (mickmister) [inline plugin/environment.go:333]: The plugin is already started and running at this point. I'm thinking "Plugin activated" may work better.

**Review by agarciamontoro — APPROVED** [review]
Oh, thank you! These two lines will be extremely useful :pray:

**Review by mickmister — APPROVED** [review]

**Review by hanzei — APPROVED** [review]
Nice one :+1:

**Comment by zeraussiul** [comment]
> This PR has been automatically labelled "stale" because it hasn't had recent activity. A core team member will check in on the status of the PR to help with questions. Thank you for your contribution!

@hanzei @agarciamontoro  is there anything else I need to do on my end?

**Comment by hanzei** [comment]
Thank you very much for the contribution @zeraussiul :+1:

**Pull request merged**